scripts/utils/logger.js

```diff
@@ -0,0 +1,95 @@
+const chalk = require('chalk');
+
+/**
+ * Terra logger for consistently formatted and decorated log, warning and error messages.
+ * Uses `chalk` module for decoration and gives direct access to chalk for complex message
+ * styling. See chalk's docs for decoration options: https://github.com/chalk/chalk.
+ */
+class Logger {
+  /**
+   * Decorates a message with the provided options.
+   * @param {string} message - The message to decorate.
+   * @param {string[]} options.decorations - The chalk decoration options (bold, underline, color, etc...).
+   */
+  static decorateText(text, decorations = []) {
+    if (!text) {
+      return '';
+    }
+
+    // Apply all chalk options to the message. Bold, Color, Underline, etc...
+    const decoratedText = decorations.reduce((acc, option) => {
+      if (chalk[option]) {
```

Comment:
Because this is now a direct pass through I think we should apply all options without the check or log a message for the invalid option. A bad option shouldn't be silent.